### ISSUE TITLE
这样做的意思在哪呢？

### ISSUE THREAD

**Issue by jesse-li** [sknightq/vue-mpc#2]
现在vue-cli提供了pages选项，多页下就没必要router了吧。

**Comment by sknightq** [comment]
Yeah, you are right. My project is based on vue-cli 2.0, which has no configurations for multiple pages. I just share my code for fun or help the people who must work on vue-cli 2.0.
For the following days, I will create a multiple pages project based on vue-cli 3.0.
What's more, thanks for your reply!
